Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: source-deps test release deploy clean
+
+VERSION ?= 1.9
```

Comment:
Shouldn't this be 1.10?

Reply:
That's a copy+paste. I assume that's the default value? If so, will change to 1.10

Reply:
Fixed